### ISSUE TITLE
Rounding of brightness values on Galactic Unicorn micropython class

### ISSUE THREAD

**Issue by Footleg** [pimoroni/pimoroni-pico#576]
Playing with the micropython examples for Galactic unicorn I noticed that brightness values show rounding errors.
```
gu.set_brightness(0.5)
print(f"Brightness set to {gu.get_brightness()}")
```
returns:
Brightness set to 0.5019608

**Comment by Gadgetoid** [comment]
Looks like it's multiplied up to an 8-bit by 256 and divided down by 255 :grimacing: 

https://github.com/pimoroni/pimoroni-pico/blob/main/libraries/galactic_unicorn/galactic_unicorn.cpp#L502-L510

**Comment by Gadgetoid** [comment]
Should hopefully be fixed by #911, floating point quantisation issues notwithstanding.

SOMEBODY CALL NINE ONE ONE! Shawty fire burning on the dance floor, whoa!
